firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -1550,13 +1705,16 @@ void loop() {
             X_commanded_movement_in_progress = false;
```

Comment:
Not related to this pull request specifically, but should `RESET` also make sure none of the axes are moving or energized?  For instance, what if someone calls `RESET` when we're in the middle of homing?  Does the axis keep moving?

Reply:
I agree with you. Laterly I would think about it.